templates/representatives/_mandates.html

```diff
@@ -0,0 +1,22 @@
+{% load representatives_tags %}
+
+{% for mandate in mandates %}
+<br/>
+
+<div id="" class="row">
+  <div class="col-sm-6 col-md-4">
+	<div class="thumbnail">
+	  <div class="caption">
+		<h3 class="text-center">
+          <a href="{{ mandate.group.get_absolute_url }}">{{ mandate.group.name }}
+            {% if mandate.group.abreviation %}{{ mandate.group.abreviation }}{% endif %}
+          </a>
+        </h3>
+		<hr>
+        {# mandate.constituency.name - #}
```

Comment:
I stumble upon this in the original template and though it could be of some significance but it's not present in the design - and then i forgot about it.